USAGE.md

```diff
@@ -136,6 +150,19 @@ try {
 }
 ```
 
+__Java__
+
+```java
+MastodonRequest<Status> request = client.statuses()
+                .postStatus("Hello World",

```

Comment:
Thanks for your contribution @grobmeier . Looking at the `.postStatus` method, I think we need to improve this on implementation side a bit to make sure, Java devs do not need to pass so many `null` values when posting a simple status. I'll have a look at this and in the meantime accept your PR.  Again, thanks for making BigBone even better.